recipes/sfml/3.0.0/conanfile.py

```diff
@@ -0,0 +1,270 @@
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "window": [True, False],
+        "graphics": [True, False],
+        "network": [True, False],
+        "audio": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "window": True,
+        "graphics": True,
+        "network": True,
+        "audio": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def build_requirements(self):
+        self.tool_requires('cmake/[>=3.24]')
+
+    def requirements(self):
+        if self.options.window:
+            # FIXME: use cci's glad
+            if self.settings.os in ["Windows", "Linux", "FreeBSD", "Macos"]:
+                self.requires("opengl/system")
+            if self.settings.os == "Linux":
+                self.requires("libudev/system")
+                self.requires("xorg/system")
+        if self.options.graphics:
+            self.requires("freetype/2.13.2")
+            self.requires("stb/cci.20230920")
+        if self.options.audio:
+            self.requires("flac/1.4.3")
+            self.requires("openal-soft/1.22.2")
+            self.requires("vorbis/1.3.7")
+            self.requires("minimp3/cci.20211201")
+
+    def validate(self):
+        if self.settings.os not in ["Windows", "Linux", "FreeBSD", "Android", "Macos", "iOS"]:
+            raise ConanInvalidConfiguration(f"{self.ref} not supported on {self.settings.os}")
+        if self.options.graphics and not self.options.window:
+            raise ConanInvalidConfiguration("sfml:graphics=True requires sfml:window=True")
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 17)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["SFML_DEPENDENCIES_INSTALL_PREFIX"] = self.package_folder.replace("\\", "/")
+        tc.cache_variables["SFML_MISC_INSTALL_PREFIX"] = os.path.join(self.package_folder, "licenses").replace("\\", "/")
+        tc.variables["SFML_BUILD_WINDOW"] = self.options.window
+        tc.variables["SFML_BUILD_GRAPHICS"] = self.options.graphics
+        tc.variables["SFML_BUILD_NETWORK"] = self.options.network
+        tc.variables["SFML_BUILD_AUDIO"] = self.options.audio
+        tc.variables["SFML_INSTALL_PKGCONFIG_FILES"] = False
+        tc.variables["SFML_GENERATE_PDB"] = False
+        tc.variables["SFML_USE_SYSTEM_DEPS"] = True
+        tc.variables["WARNINGS_AS_ERRORS"] = False
```

Comment:
```suggestion
```
This option was renamed. It's off by default though so you don't need to explicitly set it.